docs/data/README.mdx

```diff
@@ -2,10 +2,28 @@
 title: Data Overview
 ---
 
-The RPC, data indexers, Hubble, and Horizon are platforms or tools that facilitate communication between users and applications and the Stellar network. With these various platforms, users can query network data, submit transactions, and interact with smart contracts without needing to understand the low-level details of Stellar Core.
+The RPC, data indexers, analytical platforms, Galexie, Hubble, and Horizon are platforms or tools that facilitate communication between users and applications and the Stellar network. With these various platforms, users can query network data, submit transactions, and interact with smart contracts without needing to understand the low-level details of Stellar Core.
 
 This section will walk you through the differences between the various platforms and tools, what platform or tool is best for what use case, and then link to their various documentation locations.
 
+| Features               | RPC | Horizon | Galexie | Hubble |
+| ---------------------- | --- | ------- | ------- | ------ |
+| Real-time Data         | ✅  | ✅      | ✅      | ❌     |
+| Historical Data        | ❌  | ❌      | ✅      | ✅     |
+| Classic Data           | ✅  | ✅      | ✅      | ✅     |
```

Comment:
Are we still called classic `classic`?

Reply:
Also - same comment about Analytical Platforms vs Analytics Platforms mentioned here: https://github.com/stellar/stellar-docs/pull/1058#discussion_r1817252415

Reply:
Hmm I'm fine with changing it to `assets`. I don't think that encompasses all `classic` operations/data though. But maybe that's fine?

Reply:
I think it's ok for this - we can always change it if it doesn't sit right.

Reply:
updated in 0f5207ad468791a2bbf71c4e86ad11e39da74dd7